nette/notejam/app/presenters/templates/Note/edit.latte

```diff
@@ -0,0 +1,13 @@
+{block title}{$note->name}{/block}
+{block header}{$note->name}{/block}
+{block content}
+	{form editNoteForm 'class' => 'note'}
+	{label name /}
+	{input name}
```

Comment:
the same (https://github.com/komarserjio/notejam/pull/54/files#r46080442) applies for inputs and labels